fragment from /usr/src/linux-2.6.14.7/usr/include/linux/netfilter_ipv4.h
needed for frox-0.7.18 to build (using copy of libiptc from iptables 1.2)

--- linux-libc-headers-2.6.12.0/usr/include/linux/netfilter_ipv4.h	2006-04-16 22:37:08.996523576 +0300
+++ /usr/usr/include/linux/netfilter_ipv4.h	2006-04-16 22:35:28.194277687 +0300
@@ -45,4 +45,32 @@
 /* 2.4 firewalling went 64 through 67. */
 #define SO_ORIGINAL_DST 80
 
+/* IP Cache bits. */
+/* Src IP address. */
+#define NFC_IP_SRC		0x0001
+/* Dest IP address. */
+#define NFC_IP_DST		0x0002
+/* Input device. */
+#define NFC_IP_IF_IN		0x0004
+/* Output device. */
+#define NFC_IP_IF_OUT		0x0008
+/* TOS. */
+#define NFC_IP_TOS		0x0010
+/* Protocol. */
+#define NFC_IP_PROTO		0x0020
+/* IP options. */
+#define NFC_IP_OPTIONS		0x0040
+/* Frag & flags. */
+#define NFC_IP_FRAG		0x0080
+
+/* Per-protocol information: only matters if proto match. */
+/* TCP flags. */
+#define NFC_IP_TCPFLAGS		0x0100
+/* Source port. */
+#define NFC_IP_SRC_PT		0x0200
+/* Dest port. */
+#define NFC_IP_DST_PT		0x0400
+/* Something else about the proto */
+#define NFC_IP_PROTO_UNKNOWN	0x2000
+
 #endif /*__LINUX_IP_NETFILTER_H*/
